### PR TITLE
[docs] updated styles on Summary page

### PR DESCRIPTION
**Pull request by freeyoungstrong**
## What changed?

🦄...

### Definition of Done

- [ ] Dependencies checked (if applicable)
- [ ] TS types updated (if applicable)
- [ ] CHANGELOG.md updated (if applicable)
- [ ] Complete unit testing (if applicable)
- [ ] Complete screenshot testing (if applicable)
- [ ] Complete UX review (if applicable)
- [ ] Documentation design guide updated (if any)
- [ ] Documentation examples updated (if any)
- [ ] Documentation API updated (if any)
- [ ] Add link to design (if any)
- [ ] No major bugs pending
- [ ] Complete self code review

### Reviewer checklist

- [ ] Does the code work? Does it perform its intended function and the logic is correct?
- [ ] Is the code easily understandable?
- [ ] Verify the design is implemented as per the design requirements
- [ ] Verify the names used in the programs/methods/functions convey the intent and all functions commented
- [ ] Verify the unit tests are testing the code to perform the intended functionality
- [ ] Verify the unit tests cover the positive and negative scenarios
- [ ] Verify the screenshot tests are added and are they comprehensive
- [ ] Verify documentation (design/api/example) has been added/updated

### CODE REVIEW

File: website/docs/patterns/summary/examples/summary-with-minitrend.jsx [hunk 1]
```diff
@@ -127,15 +127,15 @@ const Demo = () => {
...
           </Title>
         </Tooltip>
-        <Text size={100} color="#757575" tag="p">
+        <Text size={100} color="gray-500" tag="p">
           all time
         </Text>
         <Flex alignItems="baseline" flexWrap>
-          <Text size={500} color="#4FAE33" fontWeight="bold" mr={2} tag="a">
+          <Text size={500} color="blue-400" fontWeight="bold" mr={2} tag="a">
             15%
```
Comment (phytonmk) [inline website/docs/patterns/summary/examples/summary-with-minitrend.jsx:135]: Это ведь ожидаемо что зелёный текст "15%" стало синим?

Before:
image
After:
image

File: website/docs/patterns/summary/examples/summary-with-minitrend.jsx [hunk 2]
```diff
@@ -58,15 +58,15 @@ const Demo = () => {
             Potential Organic Traffic
```
Comment (phytonmk) [inline website/docs/patterns/summary/examples/summary-with-minitrend.jsx:58]: Было бы прикольно заодно поправить текст. Выглядит немного кривова-то

image

File: website/docs/patterns/summary/examples/simple-summary.jsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 import { Box, Flex } from '@semcore/flex-box';
 import { Text } from '@semcore/typography';
 import Tooltip from '@semcore/tooltip';
 import styled from 'styled-components';
 
 const BoxBlock = styled(Box)`
   overflow: hidden;
-  border-left: 1px solid #ccc;
-  padding-left: 16px;
+  border-left: 1px solid #c4c7cf;
```
Comment (phytonmk) [inline website/docs/patterns/summary/examples/simple-summary.jsx:9]: Давай в рамках PR либо поменяем все hex цвета в примерах документации на css переменные (из `semcore/utils/style/var.css`) либо только те, которые уже были затронуты в PR

File: website/docs/patterns/summary/examples/summary-with-minitrend.jsx (hunk 1 above)
Comment (freeyoungstrong) [reply website/docs/patterns/summary/examples/summary-with-minitrend.jsx:118]: did it as in figma desigh

**Comment by lsroman** [comment]
@freeyoungstrong отличная работа 💪 Спасибо большое!

**Pull request merged**